### PR TITLE
feat(content): grenade bandolier

### PR DESCRIPTION
**Pull request by Tefnut**
## Purpose of change

Shotguns, pistols, and rifles have an over-the-shoulder bandolier. While the impact of adding an option for grenade launchers is minimal, it helps legitimize them as an option. They technically had a grenade pouch prior, but a bandolier option is something a post-cataclysm survivor would certainly think of.

## Describe the solution

I added them and a recipe, copy-pasted from rifle bandoliers with their name, description, and accepted ammo replaced. They have half the capacity due to using bulkier ammo, but I'd say that they'd be fairly balanced still.

## Describe alternatives you've considered

Not have grenade bandoliers
Buff grenade pouches slightly

## Testing

Loaded game.
Made sure they worked.
Made sure the recipe loaded.

## Additional context

While not something likely commercially sold, I feel like any post-cataclysm survivor who was lucky enough to find a grenade launcher or a stash of HEDP and the tools to make a pipe launcher would have considered a grenade bandolier.

Second PR, woo, it only took 2 tries.

### CODE REVIEW

**Comment by Tefnut** [comment]
Hopefully this time it works

**Comment by RoyalFox2140** [comment]
You bundled two PR's before anyone else comments. Details on how to revert commits in Discord.

**Comment by Tefnut** [comment]
im closing this one because i fucked up AGAIN, I'll fix it tomorrow, I'm done.

File: data/json/recipes/armor/storage.json [hunk 1]
```diff
@@ -1015,5 +1015,18 @@
...
     "components": [ [ [ "fabric_standard", 3, "LIST" ], [ "fabric_hides_any", 3, "LIST" ] ] ]
+  },
+  {
+    "result": "bandolier_grenade",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_STORAGE",
+    "skill_used": "tailor",
+    "difficulty": 3,
+    "skills_required": [ [ "gun", 1 ], [ "launcher", 1 ] ],
```
Comment (VissValdyr) [inline data/json/recipes/armor/storage.json:1026]: We removed gun and launcher requirements at 
#4522

**Pull request closed without merging**